CHANGELOG.md

```diff
@@ -3,6 +3,35 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+# 3.0.0-alpha.6 (2019-09-02)
+
+### Infrastructure:
+- chore(storybook): Fix console warnings (#116) @anicholls
+- docs(contributing): Update git guidelines to reflect new PR strategy (#109) @anicholls
+- Reduce number of manual steps required after running create-module.sh (#46) @roblevintennis
```

Comment:
Nit: I got trigger happy here - how do we feel about updating this to match the conventional commit format just for the release notes and changelog?

Related: Since we've changed our scope formatting (e.g. `TextButton` vs. `button`) do we want to standardize here? Or should we stay perfectly matched to the PR titles?

Reply:
Also we're missing https://github.com/Workday/canvas-kit/commit/eb4908c32476617c8905045f4d9446bd5abaf4d8